libr/include/r_esil.h

```diff
@@ -30,6 +30,11 @@ enum {
 	R_ESIL_OP_TYPE_REG_WRITE = 0x10,
 	R_ESIL_OP_TYPE_MATH = 0x20,
 	R_ESIL_OP_TYPE_CUSTOM = 0x40
+#if R2_590
+	,
```

Comment:
C supports trailing commas in enums, so you could move this comma to the end of previous line

Reply:
Yeah unless you hit msvc which implements their standard. It wont hurt but will be gone in a month when i pull all those r2590 changes